widget/balance/src/main/java/com/ivy/widget/balance/WalletBalanceWidgetContent.kt

```diff
@@ -125,22 +125,25 @@ fun IncomeExpenseSection(
     expense: String,
     currency: String,
 ) {
+    val resources = LocalContext.current.resources
+    val sortedIncomeCategories = sortCategoriesAlphabetically(income.split(", "))
```

Comment:
This doesn't make sense. Can you upload a screen recoding of what is the intention of this change?